core/src/main/java/be/appwise/core/extensions/GlobalExtensions.kt

```diff
@@ -1,13 +1,12 @@
 package be.appwise.core.extensions
 
 import android.view.View
-import android.widget.TextView
+import androidx.annotation.ColorRes
 import androidx.core.content.ContextCompat
 import com.google.android.material.snackbar.Snackbar
 
-@Deprecated("This is not the desired way to show/style a Snackbar, theming is easier to do globally in the themes.xml")
-internal fun snackBar(parentView: View, message: String, textColor: Int) =
-    Snackbar.make(parentView, message, Snackbar.LENGTH_LONG).apply {
-        view.findViewById<TextView>(com.google.android.material.R.id.snackbar_text)
-            .setTextColor(ContextCompat.getColor(context, textColor))
-    }
\ No newline at end of file
+fun makeSnackbar(message: String, parentView: View, @ColorRes textColor: Int?) = Snackbar.make(parentView, message, Snackbar.LENGTH_LONG).apply {
```

Comment:
I would keep the order of the parameters the same as `Snackbar.make()`

Reply:
I can see why this is requested. 
For the function `showSnackbar`, I'll keep the parameters in a different order because only the `message` is required, the other parameters have a default value.